src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -77,75 +53,75 @@ public Mapper(Map<Field, DataSource> mappings) {
 
       List<CompletableFuture<?>> futures = new ArrayList<>();
 
-      futures.add(mappings.get(Field.CREATED_BY)
-        .resolve(doc)
+      futures.add(mappings.get(Mapping.Field.CREATED_BY)
```

Comment:
I thought the point of this was to remove the hardcoded mappings in the code?

Reply:
Hardcoded mappings removed.